src/main/scala/mesosphere/marathon/api/MarathonRestModule.scala

```diff
@@ -7,7 +7,7 @@ import com.google.inject.servlet.ServletModule
 import com.google.inject.{ Provides, Scopes, Singleton }
 import mesosphere.chaos.http.HttpConf
 import mesosphere.marathon.MarathonConf
-import mesosphere.marathon.event.http.HttpEventStreamServlet
+import mesosphere.marathon.core.event.impl.stream.HttpEventStreamServlet
```

Comment:
`impl` should be a module private package. Importing it outside would violate this. Any ideas? (e.g. treat it as Servlet)